core/src/main/scala/org/apache/spark/metrics/sink/StatsdSink.scala

```diff
@@ -43,6 +44,8 @@ private[spark] class StatsdSink(
     val property: Properties, val registry: MetricRegistry) extends Sink with Logging {
   import StatsdSink._
 
+  def propertyToOption(prop: String): Option[String] = Option(property.getProperty(prop))
```

Comment:
Since this is used once, we had better remove this.

Reply:
yes makes sense